shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -2,13 +2,15 @@
     variants:
         - aarch64:
             vm_arch_name = aarch64
+        - ppc64:
```

Comment:
Seems that RHEL-8 does not have ppc64 builds.

Reply:
Yes and I don't insist on this commit. It'd just make my setup simpler.

The long story is that I'm using a single guest-os profile in order to keep using just a single vm image, be it RHEL7 or RHEL8, because I simply don't have enough disk space and I'm running el7, el7a and el8 checks within a single job. So this commit would help me in using RHEL8 profile for RHEL7 variants as well, but I agree it's ugly. If you say you want me to remove it, I will not object and will have to adjust my CI.